crutest/src/main.rs

```diff
@@ -139,7 +137,7 @@ enum Workload {
     ReplaceBeforeActive {
         /// URL location of the running dsc server
         #[clap(long, default_value = "http://127.0.0.1:9998", action)]
-        dsc: String,
+        dsc_str: String,
```

Comment:
Could we use the new global `dsc` argument here (and in `ReplaceReconcile`)?

Reply:
We can and will!  Coming  in my next PR.
It also requires some changes to the tests themselves, and I felt this PR had enough in it.